src/vm/compile.cpp

```diff
@@ -3006,29 +3006,30 @@ HRESULT NGenModulePdbWriter::WritePDBData()
 
     PEImageLayout * pLoadedLayout = m_pModule->GetFile()->GetLoaded();
 
-	// Currently DiaSymReader does not work properly generating NGEN PDBS unless 
-	// the DLL whose PDB is being generated ends in .ni.*.   Unfortunately, readyToRun
-	// images do not follow this convention and end up producing bad PDBS.  To fix
-	// this (without changing diasymreader.dll which ships indepdendently of .NET Core)
-	// we copy the file to somethign with this convention before generating the PDB
-	// and delete it when we are done.  
-	SString dllPath = pLoadedLayout->GetPath();
-	if (!dllPath.EndsWithCaseInsensitive(W(".ni.dll")) && !dllPath.EndsWithCaseInsensitive(W(".ni.exe")))
-	{
-		SString::Iterator fileNameStart = dllPath.End();
-		dllPath.FindBack(fileNameStart, DIRECTORY_SEPARATOR_STR_W);
-
-		SString::Iterator ext = dllPath.End();
-		dllPath.FindBack(ext, '.');
-
-		// m_tempSourceDllName = Convertion of  INPUT.dll  to INPUT.ni.dll where the PDB lives.  
-		m_tempSourceDllName = m_wszPdbPath;
-		m_tempSourceDllName += SString(dllPath, fileNameStart, ext - fileNameStart);
-		m_tempSourceDllName += W(".ni");
-		m_tempSourceDllName += SString(dllPath, ext, dllPath.End() - ext);
-		CopyFileW(dllPath, m_tempSourceDllName, false);
-		dllPath = m_tempSourceDllName;
-	}
+    // Currently DiaSymReader does not work properly generating NGEN PDBS unless 
+    // the DLL whose PDB is being generated ends in .ni.*.   Unfortunately, readyToRun
+    // images do not follow this convention and end up producing bad PDBS.  To fix
+    // this (without changing diasymreader.dll which ships indepdendently of .NET Core)
+    // we copy the file to somethign with this convention before generating the PDB
+    // and delete it when we are done.  
+    SString dllPath = pLoadedLayout->GetPath();
+    if (!dllPath.EndsWithCaseInsensitive(W(".ni.dll")) && !dllPath.EndsWithCaseInsensitive(W(".ni.exe")))
+    {
+        SString::Iterator fileNameStart = dllPath.End();
+        if (!dllPath.FindBack(fileNameStart, DIRECTORY_SEPARATOR_STR_W))
+            fileNameStart = dllPath.Begin();
```

Comment:
The fix is just these 2 lines, with the if-check. The rest of the changes in the file is just replacing tabs with 4 spaces